index.html

```diff
@@ -3,9 +3,20 @@
   <head>
     <title>EST UI Widget</title>
     <link rel="stylesheet" type="text/css" href="style.css">
-    <script type="text/javascript" src="javascript/est-widget.js"></script>
+
+      <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
```

Comment:
After trying many things, here is the conclusion:
[our code](https://github.com/GovWizely/est-ui-widget/pull/2/files#diff-4220180f0c119d766886877485f1806aR4) create the `script` tag and inject it correctly into `head`, but browsers do async source loading for the injected tag, and that is why in the following script, we can't use JQuery.

There are several ways on making the loading synchronous:
[1st example](https://stackoverflow.com/questions/2879509/dynamically-loading-javascript-synchronously)
[2nd example](https://stackoverflow.com/questions/8578617/inject-a-script-tag-with-remote-src-and-wait-for-it-to-execute/8578840#8578840)
but it is not recommended to do any synchronous loading.

I tried to solve it with [different ways](https://stackoverflow.com/questions/9899372/pure-javascript-equivalent-to-jquerys-ready-how-to-call-a-function-when-the), but all require the user to do some coding, or it require a fair amount of copy/paste.

What do you suggest in this case?